nucypher/characters/lawful.py

```diff
@@ -723,19 +707,14 @@ def threshold_decrypt(
         threshold_message_kit: ThresholdMessageKit,
         context: Optional[dict] = None,
         ursulas: Optional[List["Ursula"]] = None,
-        peering_timeout: int = 60,
+        decryption_timeout: int = 15,
```

Comment:
same as above, can we make this a constant value?

Reply:
✔️